tests/integration/test_workflow_yaml_validation.py

```
@@ -0,0 +1,612 @@
+
+    @pytest.fixture
+    def workflows_dir(self) -> Path:
+        """Get the workflows directory path."""
+        workflows_dir = Path(__file__).parent.parent.parent / ".github" / "workflows"
+        try:
+            # Ensure the directory exists and is accessible
+            if not workflows_dir.exists():
+                pytest.fail(f"Workflows directory not found: {workflows_dir}")
+            if not workflows_dir.is_dir():
+                pytest.fail(f"Workflows path is not a directory: {workflows_dir}")
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)